src/xbot/common/subsystems/AbstractPoseSubsystem.java

```diff
@@ -47,8 +47,7 @@
     
     private BooleanProperty rioRotated;
     
-    @Inject
-    public PoseSubsystem(WPIFactory factory, XPropertyManager propManager) {
+    public AbstractPoseSubsystem(WPIFactory factory, XPropertyManager propManager) {
```

Comment:
I'm not wildly in love with putting Abstract in the class name, I think we've tended towards `Base` in these situations so far in the commonlib?